lib/nostrum/application.ex

```diff
@@ -22,6 +24,12 @@ defmodule Nostrum.Application do
   def start(_type, _args) do
     Token.check_token!()
     check_executables()
+
+    unless Application.get_env(:nostrum, :suppress_youtubedl_version) do
+      if String.contains?(Application.get_env(:nostrum, :youtubedl, "youtube-dl"), "youtube-dl"),
+        do: check_youtubedl()
+    end
```

Comment:
am I understanding the code right that this will _always_ check for `youtube-dl` on the system, even if someone is not using voice nor not using youtube-dl, as long as they have it installed on their system? I am not sure we should automatically discover it like that. Can't we do it when nostrum is explicitly used to play audio via youtube-dl?